modules/overrides/http.go

```diff
@@ -0,0 +1,115 @@
+package overrides
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/go-kit/log/level"
+	jsoniter "github.com/json-iterator/go"
+	"github.com/opentracing/opentracing-go"
+	"github.com/pkg/errors"
+	"github.com/weaveworks/common/user"
+
+	"github.com/grafana/tempo/pkg/api"
+	"github.com/grafana/tempo/pkg/util/log"
+)
+
+// OverridesHandler is a http.HandlerFunc to return user configured overrides
+func (o *userConfigOverridesManager) OverridesHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	span, ctx := opentracing.StartSpanFromContext(ctx, "userConfigOverridesManager.OverridesHandler")
+	defer span.Finish()
+
+	userID, err := user.ExtractOrgID(ctx)
+	if err != nil {
+		handleError(r, w, userID, errors.Wrap(err, "failed to find org id in request"))
+		return
+	}
+	level.Info(log.Logger).Log("tenant", userID, "method", r.Method, "url", r.URL.RequestURI())
+
+	switch r.Method {
```

Comment:
we should use the mux to handle this routing

Reply:
Good point, I've split up the handler and register them separately now in modules.go. It's a bit more verbose but not more code overall I think.